server/app.py

```diff
@@ -54,11 +55,27 @@
     notify_slack,
     send_email_new_business_membership,
     send_multiple_account_warning,
+    name_splitter,
 )
 
 ZONE = timezone(TIMEZONE)
 USE_THERMOMETER = False
 
+SUBSCRIPTION_TYPE = {
+    "Texas Tribune Sustaining Membership": {
+        "type": "Sustaining Membership",
+    },
+    "Texas Tribune Business Membership": {
+        "type": "Business Membership",
+    },
+    "Texas Tribune Circle Membership": {
+        "type": "Giving Circle",
+    },
+    "Blast Subscription": {
+        "type": "The Blast",
+    },
+}
+
```

Comment:
Ahh, good question. That mapping isn't used in this pr but it will be used for our /business, /circle and /blast prs to suss out the opportunity type from the stripe description

Reply:
gotcha, we'll circle back to it then 😉